src/test/java/com/google/spanner/liquibase/LiquibaseTests.java

```diff
@@ -206,7 +206,6 @@ void doDropAllForeignKeysTest(TestHarness.Connection testHarness) throws Excepti
     }
   }
 
-  @Disabled("The emulator erroneously requires the optional COLUMN keyword to be included in ALTER TABLE <table> ADD [COLUMN] ...")
```

Comment:
Version 1.3.0 of the emulator that just got release has fixed this problem.

Reply:
Hmm... The build on CircleCI consistently times out when using the newest version of the emulator. Using the previous version works without problems. So reverting this change for now.